cualid/tests/test_mint.py

```diff
@@ -16,6 +16,11 @@ def test_create_ids_fail(self):
         self.assertTrue(len(result) > 0)
         self.assertTrue(len(result) < 1000)
 
+    def test_create_ids_API(self):
+        result = list(create_ids(100, 7))
+        self.assertTrue(len(str(result[0][0])) < 10)
+        self.assertTrue(len(str(result[0][1])) > 10)
```

Comment:
a very minor comment, but will you change these to use the `==` operator rather than the greater than less than for the cualid and then check the type for the uuid?

```
id_length = 7
result = list(create_ids(100, id_length))
self.assertTrue(len(result[0][0]) == id_length)
self.assertTrue(type(result[0][1]) == uuid.UUID)
```